engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -43,6 +42,7 @@ object SlickDataAccess {
   private lazy val rootDatabaseConfig = rootConfig.getConfig("database")
   private lazy val databaseConfigName = rootDatabaseConfig.getStringOption("config")
   lazy val defaultDatabaseConfig = databaseConfigName.map(getDatabaseConfig).getOrElse(rootDatabaseConfig)
+  val actorSystem = ActorSystem("slick-retry-scheduler")
```

Comment:
Before you flog me... I wasn't sure what to do about this.  In order to use `akka.pattern.after` in [line 209](https://github.com/broadinstitute/cromwell/pull/642/files#diff-e3d71de750517c4e29193d05735f0cdcR209), I need a scheduler and I wasn't able to create a scheduler manually, it seemed like the way to get one was from an `ActorSystem`, so I just created this one here just so I could get this working.  It was non-obvious to me how to get the main `ActorSystem` piped in here because currently.  I could use `CromwellServer.actorSystem` instead, but again, I'm not sure.